core-bundle/src/Resources/contao/languages/en/tl_image_size.xlf

```diff
@@ -59,6 +59,9 @@
       <trans-unit id="tl_image_size.formatsMissingWebp">
         <source>You can enable WEBP images by adding "webp" to &lt;em&gt;contao.image.valid_extensions&lt;/em&gt;.</source>
       </trans-unit>
+      <trans-unit id="tl_image_size.formatsWebpNotSupported">
+        <source>WEBP is not supported on your platform.</source>
```

Comment:
We should probably say "server" instead of "platform", shouldn't we? It is more "normal user" friendly. 🤣

Reply:
Done 6538233895b61dc3ce5e3bcda5e816312546c2f8